scripts/example.ts

```diff
@@ -120,11 +149,62 @@ const addPackageScripts = async ({
   await writeJSON(packageJsonPath, packageJson, { spaces: 2 });
 };
 
+async function readMainConfig({ cwd }: { cwd: string }) {
+  const configDir = path.join(cwd, '.storybook');
+  const mainConfigPath = getInterpretedFile(path.resolve(configDir, 'main'));
+  return readConfig(mainConfigPath);
+}
+
+// NOTE: the test here will apply whether the path is symlink-preserved or otherwise
+const webpackFinalCode = `
+  (config) => ({
+    ...config,
+    module: {
+      ...config.modules,
+      rules: [
+        {
+          test: [/\\/node_modules\\/@storybook\\/[^/]*\\/template\\/stories\\//],
+          loader: '../../code/node_modules/esbuild-loader',
```

Comment:
We should be able to require.resolve
And thus get a fully resolved path generated.